### ISSUE TITLE
Parentheses for JPEGSaveOptions needed

### ISSUE THREAD

**Issue by dagayaru** [shotgunsoftware/tk-photoshopcc#65]
```
2019-04-02 22:58:45,430 [19008 WARNING sgtk.env.project.tk-photoshopcc] Couldn't generate thumbnail: 'properties'
Traceback (most recent call last):
  File "...\AppData\Roaming\Shotgun\bundle_cache\app_store\tk-photoshopcc\v1.7.1\engine.py", line 454, in generate_thumbnail
    quality=3, # Default quality value for Photoshop Jpeg option
  File "...\AppData\Roaming\Shotgun\bundle_cache\app_store\tk-photoshopcc\v1.7.1\engine.py", line 403, in export_as_jpeg
    jpeg_options.quality = quality
  File "...\AppData\Roaming\Shotgun\bundle_cache\app_store\tk-framework-adobe\v1.0.2\python\tk_framework_adobe\rpc\proxy.py", line 278, in __setattr__
    remote_names = self.data["properties"] + self.data["methods"].keys()
KeyError: 'properties'

```

https://github.com/shotgunsoftware/tk-photoshopcc/blob/b70f624a36f60f8e7fa2e32f7469d902f4ce792b/engine.py#L402

**Comment by willis102** [comment]
Created internal ticket SG-11929, and [PR](https://github.com/shotgunsoftware/tk-photoshopcc/pull/66).

**Comment by willis102** [comment]
Fixed in v1.7.2
